src/js/containers/agencyV2/visualizations/RecipientDistributionContainer.jsx

```diff
@@ -49,6 +51,7 @@ const RecipientDistributionContainer = ({ fiscalYear }) => {
             .then((res) => {
                 const recipientDistribution = Object.create(BaseAgencyRecipients);
                 recipientDistribution.populate(res.data);
+                setData(res.data);
```

Comment:
if we are setting this agency data in redux then we should retrieve that data via redux in the recipient distribution object. We do not need the state data property.

Reply:
Updated